test/e2e/azure_test.go

```diff
@@ -66,3 +67,9 @@ func (c AzureCloudAssert) HasPodVM(t *testing.T, id string) {
 		t.Error("PodVM was not created")
 	}
 }
+
+// Func CmdsToCheckSEPodVM need to be implemented in Azure.
+func (c AzureCloudAssert) CmdsToCheckSEPodVM(pod *corev1.Pod) []testCommand {
```

Comment:
It's not reasonable to have a function CmdsToCheckSEPodVM in azure conditional branch as there is no SE in azure.

Reply:
azure have TEE support https://learn.microsoft.com/en-us/azure/confidential-computing/overview Didn't find whether  they can support peer-pod.

Reply:
Updated